.github/workflows/main.yml

```
@@ -44,8 +44,9 @@ jobs:
       run: |
         mkdir llvm_src/build
         cd llvm_src/build
+        # Debug mode is needed for the pattern search index
         cmake ../llvm \
-          -DCMAKE_BUILD_TYPE=Release \
+          -DCMAKE_BUILD_TYPE=Debug \
```

We actually don't need DEBUG but Asserts I believe:

```suggestion
          -DCMAKE_BUILD_TYPE=Release \
          -DLLVM_ENABLE_ASSERTIONS=ON \
```